wiki/Article_Style_Guide/en.md

```diff
@@ -65,34 +105,63 @@ Note that heading levels 3 and onwards use the hashtag style.
 
 Titles are to be in title case.
 The title of an article is the name of the folder that the article is located in.
+
 For English article titles, if you need to reword the title, you must rename the folder to match the article title.
 
 All article titles are to be using the level 1 heading.
+For example:
+
+```
+# Folder Name
+```
+
+#### Untranslatable Titles
+
+In the case where a title can **not** be translated, use the language title (in English) followed by the language name (in that language) in round brackets.
+Here are some examples:
+
+```
+# osu!mania (Español)
+# Live Streaming (日本語)
+# Skinning osu!catch (Français)
+```
+
+You can find a list of native language names at [Language names in their own languages and scripts](http://www.omniglot.com/language/names.htm) or [List of language names](https://en.wikipedia.org/wiki/List_of_language_names).
+<!-- NOTE this may introduce more inconsistencies because some languages use more than one script to represent their language! -->
 
 ### Section Headings
 
 All section headings are to use title case, just like the article titles.
 
-All section headings are to be using level 2 to level 5 headings.
+All section headings are to be using levels 2 to 5 headings.
 
 Section headings must **never** have a link in it.
 Instead place the link underneath the section heading.
 For example:
 
 ```
-## Editor
+## Beatmap Editor
 
 For a full explanation, see [Beatmap Editor](/wiki/Beatmap_Editor/).
 ```
 
+Section headings must **never** have any styles applied to it.
+
+Section headings can have **small icons** in their section headings but never images larger than 1 point (the typical font size height; usually 16 pixels).
+For example:
+
+```
+#### ![osu! icon](/wiki/shared/Osu.gif) osu!
+```
+
 ## TOC
 
 The TOC (Table of Contents) will automatically be generated on the osu!wiki.
-Because of this, there is no need to include add one.
+Because of this, there is no need to add one.
```

Comment:
would this do? `ToCs (Table of Contents) are automatically generated in the osu!web, because of this do not add one.`
... `no longer a requirement` sounds like it is optional